display_information/display-dimensions/src/main/java/com/esri/samples/display_dimensions/DisplayDimensionsSample.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright 2020 Esri.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.esri.samples.display_dimensions;
+
+import com.esri.arcgisruntime.layers.DimensionLayer;
+import com.esri.arcgisruntime.layers.Layer;
+import com.esri.arcgisruntime.loadable.LoadStatus;
+import com.esri.arcgisruntime.mapping.MobileMapPackage;
+import com.esri.arcgisruntime.mapping.Viewpoint;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import javafx.application.Application;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Alert;
+import javafx.scene.control.CheckBox;
+import javafx.scene.control.Label;
+import javafx.scene.layout.*;
+import javafx.scene.paint.Paint;
+import javafx.stage.Stage;
+
+import java.io.File;
+import java.util.Objects;
+
+public class DisplayDimensionsSample extends Application {
+
+  private DimensionLayer dimensionLayer;
+  private MapView mapView;
+  private MobileMapPackage mobileMapPackage;
```

Comment:
Is the mmpk a member variable due to garbage collection? Or can it be declared locally?

Reply:
for garbage collection yeh: I'll add a comment!